sdk/postgresql/Azure.ResourceManager.PostgreSql/src/PostgreSqlFlexibleServers/Generated/Models/PostgreSqlFlexibleServerCapabilityProperties.cs

```diff
@@ -16,34 +16,36 @@ public partial class PostgreSqlFlexibleServerCapabilityProperties
         /// <summary> Initializes a new instance of PostgreSqlFlexibleServerCapabilityProperties. </summary>
         internal PostgreSqlFlexibleServerCapabilityProperties()
         {
+            SupportedHAModes = new ChangeTrackingList<string>();
             SupportedFlexibleServerEditions = new ChangeTrackingList<PostgreSqlFlexibleServerEditionCapability>();
             SupportedHyperscaleNodeEditions = new ChangeTrackingList<PostgreSqlFlexibleServerHyperscaleNodeEditionCapability>();
-            SupportedHAModes = new ChangeTrackingList<string>();
         }
 
         /// <summary> Initializes a new instance of PostgreSqlFlexibleServerCapabilityProperties. </summary>
         /// <param name="zone"> zone name. </param>
+        /// <param name="supportedHAModes"> Supported high availability mode. </param>
         /// <param name="isGeoBackupSupported"> A value indicating whether a new server in this region can have geo-backups to paired region. </param>
         /// <param name="isZoneRedundantHASupported"> A value indicating whether a new server in this region can support multi zone HA. </param>
         /// <param name="isZoneRedundantHAAndGeoBackupSupported"> A value indicating whether a new server in this region can have geo-backups to paired region. </param>
         /// <param name="supportedFlexibleServerEditions"></param>
         /// <param name="supportedHyperscaleNodeEditions"></param>
-        /// <param name="supportedHAModes"> Supported high availability mode. </param>
         /// <param name="status"> The status. </param>
-        internal PostgreSqlFlexibleServerCapabilityProperties(string zone, bool? isGeoBackupSupported, bool? isZoneRedundantHASupported, bool? isZoneRedundantHAAndGeoBackupSupported, IReadOnlyList<PostgreSqlFlexibleServerEditionCapability> supportedFlexibleServerEditions, IReadOnlyList<PostgreSqlFlexibleServerHyperscaleNodeEditionCapability> supportedHyperscaleNodeEditions, IReadOnlyList<string> supportedHAModes, string status)
```

Comment:
This is a breaking change. I saw the new version is `1.1.0-beta1`, so it is OK. But you only change the sequence of the parameter, which should be avoided.

Reply:
@alanenriqueo I just realized that there is already an official release `1.0.0`. So this is a breaking change which needs to be resolved.

Here are 2 options:
  - Add manual codes to provide backward compatibility, see compute for reference: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/compute/Microsoft.Azure.Management.Compute/src/Customizations
- Or you need to bump the major version, e.g. `2.0.0`

Reply:
Changing order of properties is no longer a brekaing change in track 2 SDKs